homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -111,9 +113,15 @@ def state(self):
         # key Fuel_type is not available when the fuel station is closed, use "get" instead of "[]" to avoid exceptions
         return self._coordinator.data[self._station_id].get(self._fuel_type)
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique, friendly identifier for this entity."""
+        return f"{self._station_id}_{self._fuel_type}"
+
     @property
     def device_state_attributes(self):
         """Return the attributes of the device."""
+
```

Comment:
Is this empty line needed? Feels inconsistent with the "unique_id" method you are adding above.
```suggestion

```